app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
```

Comment:
Yeah that sounds like a plan to me (we may still want DB-level notes of inbound vs outbound for query purposes but we can always add that easily when / if needed)